elliottlib/bzutil.py

```diff
@@ -635,6 +660,9 @@ def get_tracker_bugs(self, bug_ids: List, strict: bool = False, verbose: bool =
     def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
         return [b for b in self.get_bugs(bug_ids, permissive=not strict, verbose=verbose) if b.is_flaw_bug()]
 
+    def filter_attached_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = False):
+        return [b.id for b in self.get_bugs(bug_ids, permissive=not strict, verbose=verbose) if not b.is_attached()]
```

Comment:
Did you try this? I think `@property` means that the behavior is not called as a method with `()`, but as a property.

Reply:
updated